runtime/src/iree/hal/drivers/vulkan/native_executable.cc

```diff
@@ -29,6 +29,10 @@ using namespace iree::hal::vulkan;
 typedef struct iree_hal_vulkan_entry_point_t {
   VkPipeline pipeline;
   iree_string_view_t name;
+
+  // Optional debug information.
```

Comment:
guard this with IREE_TRACE like on the CUDA side

Reply:
Okay, will do. I tried that (https://github.com/openxla/iree/pull/13333/commits/00e94f5bccf7a00a801003aac5fdffcd40def6ad) but it felt weird having `iree_hal_vulkan_native_executable_entry_point_source_location` return partial information when tracing was disabled